SharedPackages/BrowserServicesKit/Sources/NetworkProtection/Status/NetworkProtectionStatusReporter.swift

```diff
@@ -23,6 +23,7 @@ import Common
 /// Classes that implement this protocol are in charge of relaying status changes.
 ///
 public protocol NetworkProtectionStatusReporter {
+    var vpnEnabledObserver: VPNEnabledObserver { get }
```

Comment:
The VPN-enabled status is a new concept.  Previously we were looking at connection status to set the VPN toggle status... now we're looking at connection status + on-demand status.

This observer provides access to that.